src/components/feedback/index.tsx

```diff
@@ -154,8 +223,26 @@ export const Feedback = ({mode, tripPattern, quayListData}: FeedbackProps) => {
     setSubmitted(false);
   }, [tripPattern]);
 
-  if (!category) return null;
   if (submitted) return <SubmittedComponent />;
+  if (!category) return null;
+
+  if (!displayStats || displayStats.length < 1) return null;
+  const statsForCurrentVersion = displayStats.find(
+    (entry) => entry.version === APP_VERSION,
+  );
+  if (!statsForCurrentVersion) return null;
```

Comment:
This check is present because `statsForCurrentVersion` is a `find()` on the variable `displayStats` that is set in `getDisplayStatsFromStorage()` which runs in an asynchronous `useEffect`. 

This means that `statsForCurrentVersion` will be undefined when the component mounts, until the `useEffect` has finished. During this period we do not want the Feedback component to render.